### PR TITLE
updated types, downcase translation map, updated tests, closes #92

### LINKED ISSUE #92: review + repair 3D / Objects split among object types
Europeana to DCMI to MODS to MARC types mappings for `cho_type` and `cho_edm_type` aren't clear on handling objects (3d objects) vs digital representation types (i.e. 3d files).

Depending on decision, need to update translationMapping currently waiting in #65 and fix some config mappings.

### PR DESCRIPTION
**Pull request by cmh2166**

### CODE REVIEW

File: lib/traject/macros/xml.rb [hunk 1]
```diff
@@ -21,6 +21,11 @@ def apply_translation(values, options)
       translation_map = Traject::TranslationMap.new(options[:translation_map])
       # without overwriting (further) translation map, could add
       # fuzzy match method here after pulling array out of TM
+      if values.is_a? Array
```
Comment (jcoyne) [inline lib/traject/macros/xml.rb]: How about condensing this to `values = Array(values).map(&:downcase)`

File: lib/traject/macros/xml.rb [hunk 2]
```diff
@@ -21,6 +21,11 @@ def apply_translation(values, options)
...
+        values = values.map(&:downcase)
+      elsif values.is_a? String
+        values = value.downcase
+      end
       translation_map.translate_array! values
```
Comment (cbeer) [inline lib/traject/macros/xml.rb]: Can the value be a string? I thought all these traject things were arrays.

(Also, `translate_array!` below can/should probably be `translate_array`? )

File: lib/traject/macros/xml.rb (hunk 1 above)
Comment (cmh2166) [inline lib/traject/macros/xml.rb]: sure, sounds good

File: lib/traject/macros/xml.rb (hunk 2 above)
Comment (cmh2166) [inline lib/traject/macros/xml.rb]: some can be a string

File: lib/traject/macros/xml.rb (hunk 1 above)
Comment (cmh2166) [inline lib/traject/macros/xml.rb]: done.

**Pull request merged**